openshift/tests-extension/test/qe/specs/olmv1_cc.go

```diff
@@ -34,4 +43,1061 @@ var _ = g.Describe("[sig-olmv1][Jira:OLM] clustercatalog", g.Label("NonHyperShif
 
 	})
 
+	g.It("PolarionID:69123-[Skipped:Disconnected]Catalogd clustercatalog offer the operator content through http server", g.Label("LEVEL0"), func() {
+		var (
+			baseDir                = exutil.FixturePath("testdata", "olm")
+			clustercatalogTemplate = filepath.Join(baseDir, "clustercatalog.yaml")
+			clustercatalog         = olmv1util.ClusterCatalogDescription{
+				Name:     "clustercatalog-69123",
+				Imageref: "quay.io/olmqe/olmtest-operator-index:nginxolm69123",
+				Template: clustercatalogTemplate,
+			}
+		)
+		g.By("Create clustercatalog")
+		defer clustercatalog.Delete(oc)
+		clustercatalog.Create(oc)
```

Comment:
```suggestion
		unique := rand.String(4)
		catName := "catalog-" + unique
		imageRef := "quay.io/olmqe/olmtest-operator-index:nginxolm69123"

		By("creating catalog with ....") // WHAT 69123 means? 
		cleanup, err := helpers.CreateClusterCatalog(ctx, catName, imageRef)
		Expect(err).NotTo(HaveOccurred(), "failed to create ClusterCatalog")
		DeferCleanup(cleanup)
```

I noticed we already have helpers to create and clean up Cluster Catalogues with an image: https://github.com/openshift/operator-framework-operator-controller/blob/main/openshift/tests-extension/pkg/helpers/cluster_catalog.go#L24-L52

Could we please use those helpers here instead?
It’ll help reduce extra dependencies and make the code easier to maintain long term.
(When we spread similar logic in different places, it gets harder to keep things consistent.)

Example for reference:
🔗 https://github.com/openshift/operator-framework-operator-controller/blob/0e8f138d8f87e48955e337ecffa691905aa5be11/openshift/tests-extension/test/olmv1-catalog.go#L205-L213

Thanks a lot 🙏